src/components/concept-page/concept-page.component.jsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import image from '../../assets/concept_bg.png'
+
+import './concept-page.styles.scss';
+
+const ConceptPage = () => (
+    <div className="full-container">
+
+        <div className="page-break" />
+        <div className="page-container">
+
+            <div className="title">DEAR COCO is...</div>
+            <div className="subheading">
+                オーガニックココナッツから生まれた
+                <br />
+                <br />
+                地球と人に優しいコスメブランドです
+            </div>
+
+            <div className="two-halfs-container">
+
+                <div className="left-half-container">
+
+                    <div className="textbox">
+                        生命の木とも呼ばれるココヤシの木。
+                        <br />
+                        <br />
+                        その実から採れるココナッツオイルは栄養豊富で、
+                        <br />
+                        <br />
+                        美容と健康に多くの利益をもたらします。
+                        <br />
+                        <br />
+                        DEAR COCOは、100%オーガニック栽培の
+                        <br />
+                        <br />
+                        タイ産ココナッツのみを厳選。
+                        <br />
+                        <br />
+                        <br />
+                        <br />
+                        こだわりのココナッツオイルと
+                        <br />
+                        <br />
+                        自然由来の成分のみを使用した
+                        <br />
+                        <br />
+                        地球と人に優しいコスメブランドです。
+                    </div>
```

Comment:
you should probably use css to style the text. use line height to control spacing between 2 lines, and there should be another property to control the spacing between paragraphs

Reply:
Needed to keep one of the breaks per line because they the lines/sentences are different lengths, but deleted the others and used line height as well as separating the paragraphs into 2 elements